docs/content/integrations/embedded-elt.mdx

```diff
@@ -0,0 +1,126 @@
+
+You can provide either a connection string or a dictionary of connection parameters to each of these classes.
+
+For details on what connection parameters Sling accepts for each integration, see the <a href="https://docs.slingdata.io/connections/database-connections">Sling Connections</a> page.
+
+To create an Asset that syncs between two connections, you can use the <PyObject module="dagster-embedded-elt.sling" object="build_sling_asset" /> factory.
+
+---
+
+## Setting up a Sling Resource
+
+A Sling Resource is a Dagster Resource that contains references to both a Source Connection and a Target Connection. Sling is versatile in what a source or destination can represent. You can provide arbitrary keywords to the <PyObject module="dagster-embedded-elt.sling" object="SlingSourceConnection" /> and <PyObject module="dagster-embedded-elt.sling" object="SlingTargetConnection" /> classes.
+
+The types and parameters for each connection are defined by <a href="https://docs.slingdata.io/connections/database-connections">Sling's Connections</a>.
+
+The simplest connection is a file connection, which can be defined as:
+
+```python
+from dagster_embedded_elt.sling import SlingSourceConnection
+source = SlingSourceConnection(type="file")
+```
+
+Note that no path is required, as that is provided by the asset itself.
+
+```python
+asset_def = build_sling_asset(
+    asset_spec=AssetSpec("my_file"),
+    source_stream=f"file://{path_to_file}",
+    ...
+)
+
+```
+
+For database connections, you can provide a connection string or a dictionary of keyword arguments. For example, to connect to a SQLite database, you can provide an path to the database using the `instance` keyword, which is specified on the <a href="https://docs.slingdata.io/connections/database-connections/sqlite">Sqlite Connection</a> page.
+
+```python
+source = SlingSourceConnection(type="sqlite", instance="path/to/sqlite.db")
+```
+
+Here are some additional examples of database connections:
+
+```python
+source = SlingSourceConnection(
+    type="postgres", host="localhost", port=5432, database="my_database",
+    user="my_user", password=EnvVar("PG_PASS")
+)
+
+source = SlingSourceConnection(
+    type="snowflake", host="hostname.snowflake", user="username",
+    database="database", password=EnvVar("SF_PASSWORD"), role="role"
+)
+```
+
+Similarily, you can define file/storage connections:
+
+```python
+source = SlingSourceConnection(
+    type="s3", bucket="sling-bucket",
+    access_key_id=EnvVar("AWS_ACCESS_KEY_ID"),
+    secret_access_key=EnvVar("AWS_SECRET_ACCESS_KEY")
+)
+
+```
+
+## Creating a Sling Sync
+
+To create a Sling Sync, once you have defined your Resource, you can use the <PyObject module="dagster_embedded_elt.sling" object="build_sling_asset" /> factory to create an Asset.
```

Comment:
```suggestion
To create a Sling sync, once you have defined your resource, you can use the <PyObject module="dagster_embedded_elt.sling" object="build_sling_asset" /> factory to create an asset.
```

These are not proprietary terms for us (I don't think Sling capitalizes `sync` either). As for `asset`, only `Software-defined Asset(s)` needs to be capitalized